packaging/os/yumrepo.py

```diff
@@ -134,6 +174,14 @@
         are allowed. Substitution variables (e.g. C($releasever)) are honored
         here.
       - The list can also be a regular YAML array.
+  ip_resolve:
```

Comment:
The `whatever` is a keyword used in YUM to deal with the IP resolving. We can not change this here.